pytorch_lightning/trainer/trainer.py

```diff
@@ -132,7 +132,7 @@ def __init__(self,
         self.gradient_clip_val = gradient_clip_val
         self.check_val_every_n_epoch = check_val_every_n_epoch
         self.track_grad_norm = track_grad_norm
-        self.on_gpu = gpus is not None and torch.cuda.is_available()
+        self.on_gpu = True if (gpus and torch.cuda.is_available()) else False
```

Comment:
this is  the same as:
```python
self.on_gpu = gpus and torch.cuda.is_available()
```

Reply:
That also was the first comment of @Borda (who deleted it I guess?) and I answered to that. 
It is different, because the way I propose, `self.on_gpu` is always a boolean.
In the way you suggest, `self.on_gpu` will either be a boolean, an integer or a list. 
Am I missing something? ^^

Reply:
no, it should be a boolean. i wrote this before the table below